src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -1073,7 +1058,8 @@ void M()
             // Trigger background analysis to ensure analyzer diagnostics are computed and cached. 
             // We enable full solution analysis so the 'AnalyzeDocumentAsync' doesn't skip analysis based on whether the document is active/open.
             workspace.GlobalOptions.SetGlobalOption(SolutionCrawlerOptionsStorage.BackgroundAnalysisScopeOption, LanguageNames.CSharp, BackgroundAnalysisScope.FullSolution);
-            await diagnosticIncrementalAnalyzer.AnalyzeDocumentAsync(sourceDocument, bodyOpt: null!, InvocationReasons.DocumentChanged, CancellationToken.None);
+
+            await diagnosticIncrementalAnalyzer.ForceAnalyzeProjectAsync(sourceDocument.Project, CancellationToken.None);
```

Comment:
AnalzyeDocument went away, but we stilll have ForceAnalyzeProject (called when the user invokes "Run code analysis on project).  So these tests could remain, just using that entrypoint to help simulate the behavior here.